### ISSUE TITLE
Provide a Gradio Demo 

### ISSUE THREAD

**Issue by MohamedAliRashad** [snap-research/NeROIC#7]
It will be nice if we can try out the model on huggingface using gradio

**Comment by Eman2022** [comment]
I agree that would be very nice- especially seeing that windows is not supported if we train it ourselves

**Comment by huyyxy** [comment]
Here is a document detailing a successful run on Tencent Cloud servers, although it is transcribed in Chinese.

https://cloud.tencent.com/developer/article/2347349
